xep-0369.xml

```diff
@@ -80,11 +85,11 @@
 
 <section1 topic='Requirements' anchor='reqs'>
   <ul>
-    <li>A user's participation in a channel persists and is not modified by the user's client going online and offline.</li>
+    <li>A user's participation in a channel may persist and is not modified by the user's client going online and offline.</li>
```

Comment:
It is required, isn't it? We don't introduce a mode for doing MUC-style 'disconnections eject you' explicitly?